src/components/Hamburger/_hamburger.scss

```diff
@@ -8,11 +8,11 @@
     color: var(--colour-white);
   }
 
-  &[disabled] {
+  @include mq(md) {
     display: none;
   }
 
-  @include mq(md) {
+  &[disabled] {
```

Comment:
I reckon this qualifies as a state so should be added to the rule for .is-\.has

Reply:
👍